### PR TITLE
DOC Update history viewer GraphQL examples for CMS 5

### PR DESCRIPTION
**Pull request by GuySartorelli**
Following on from https://github.com/silverstripe/developer-docs/pull/211, removes mention of GraphQL v3 and updates javascript to work with the CMS 5 dependencies.

## Issue
- https://github.com/silverstripe/developer-docs/issues/62

### CODE REVIEW

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md [hunk 1]
```diff
@@ -955,18 +930,25 @@ Injector.ready(() => {
 
   // Assuming you've got some element in the DOM with the id "notes-app"
   $('#notes-app').entwine({
+    ReactRoot: null,
+
     onmatch() {
-      ReactDOM.render(
+      const root = createRoot(this[0]);
```
Comment (sabina-talipova) [inline en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md]: Just question, what do we have in `this[0]` exactly? Probably we could add a short comment to explain this in code snippet.

Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md]: `this[0]` is a fairly common way in jQuery to say "Give me the underlying node that this jQuery object represents so I can use it with non-jQuery code". I'll add a comment when you've done with your review and assigned it back.

**Comment by sabina-talipova** [comment]
Looks good. Could you add comments for this, please.

**Comment by GuySartorelli** [comment]
@sabina-talipova Do you remember what comments you wanted on this?

**Comment by sabina-talipova** [comment]
> @sabina-talipova Do you remember what comments you wanted on this?

In this comment you mentioned that you will update comment for `this`. https://github.com/silverstripe/developer-docs/pull/196#discussion_r1149742605

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md (hunk 1 above)
Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md]: This part is no longer in this PR - see https://github.com/silverstripe/developer-docs/pull/247 instead.

**Review by sabina-talipova — APPROVED** [review]
LGFM. Tested locally. Provided example works.

**Pull request merged**